src/cmdlib.sh

```diff
@@ -247,18 +247,32 @@ prepare_build() {
     mkdir tmp && TMPDIR=$(pwd)/tmp
 }
 
+commit_overlay() {
+    local name path respath
+    name=$1
+    path=$2
+    respath=$(realpath "${path}")
+    echo -n "Committing ${name}: ${path} ... "
+    ostree commit --repo="${tmprepo}" --tree=dir="${respath}" -b "${name}" \
+        --owner-uid 0 --owner-gid 0 --no-xattrs --no-bindings --parent=none \
+        --timestamp "${git_timestamp}"
```

Comment:
Hmm, `git_timestamp` is a local var in the other function though, no? Although `ShellCheck` isn't complaining; am I missing something?

Reply:
The answer basically is [we need to stop writing in shell script](https://unix.stackexchange.com/questions/282557/scope-of-local-variables-in-shell-functions)...

Reply:
On this topic, https://timryan.org/2018/07/02/moving-from-the-shell-to-rust-with-commandspec.html is interesting...